README.md

```diff
@@ -3,7 +3,7 @@ This is not an officially supported Google product
 
 **Coathored by:** Alexei Czeskis, Thai Duong, Eduardo' Vela'' \<Nava\>, and Adam Stubblefield.
 
-**Status:** Implemented in Java (aczeskis@google.com)
+**Status:** Implemented in Java and C++ (aczeskis@google.com)
```

Comment:
actually, maybe just revert this line here and add it in the C++ PR?  you have it correct in that PR. :)

Reply:
I updated the readme, and re-run tests once again.
they pass, with an obvious exception of java/c++ interworking code which can not possibly work without c++ code.
It will work when both PR4 and PR5 are merged